prov/verbs/src/fi_verbs.h

```diff
@@ -127,11 +126,26 @@ struct verbs_dev_info {
 	struct dlist_entry addrs;
 };
 
+struct fi_ibv_device_info {
+	struct dlist_entry list_entry;
+	struct fi_info *info_msg;
+	struct fi_info *info_rdm;
+	struct ibv_port_attr *port_attr;
+	struct ibv_device_attr *device_attr;
```

Comment:
I suppose the need for this list would go away once we start using the max_ep_srx_ctx attribute directly, right?

Reply:
Yes, we have a correct reflection of the device/port attributes to the provider's `fi_info` attributes.
Seems, we can drop these changes.
But I'm going to remove port_attr/device_attr from this structure. Because I guess, I could return back during re-implementation of verbs/UD. Anyway, it can be simply extended by these parameters

Reply:
Sorry, I missed to respond to this comment. If this list is needed for verbs/UD later, can you add this change in a separate patch/PR when it's actually required? I don't see how it's useful now.